### ISSUE TITLE
Publish to NPM

### ISSUE THREAD

**Issue by mike183** [ember-engines/engine-blueprint#10]
Are there any updates on when this will be published to the NPM registry?

cc/ @rwjblue @trentmwillis

**Comment by trentmwillis** [comment]
Does it need to be published? I thought the blueprint functionality from ember-cli could pull from the repo

**Comment by rwjblue** [comment]
Yep, it can, but it’s a bit nicer to say `ember new my-engine -b @ember-engines/blueprint` vs `ember new my-engine -b git@github.com:ember-engines/engine-blueprint.git`...

**Comment by mike183** [comment]
As @rwjblue said, its easier to type `ember new my-engine -b @ember-engines/blueprint`.

In addition, I'd also say that it was easier to type from memory.

**Comment by trentmwillis** [comment]
Ah, I assumed there would be a shorthand for GitHub lookups. Also, I think I’m lazy and just plan on copy-pasting no matter which route is chosen 😛

**Comment by mike183** [comment]
Is there anything blocking it being published? Or is it just a case of whoever has permission publishing it?
